state/factory/workingset.go

```diff
@@ -117,6 +119,9 @@ func (ws *workingSet) runActions(
 			return nil, errors.Wrap(err, "error when run action")
 		}
 		receipts = append(receipts, receipt)
+		if fCtx.EnableDynamicFeeTx && blkCtx.BaseFee != nil {
+			(&blkCtx.AccumulatedTips).Add(&blkCtx.AccumulatedTips, new(big.Int).Sub(receipt.EffectiveGasPrice, blkCtx.BaseFee))
```

Comment:
this is the price, need to multiply Gas()?

Reply:
also, is it possible to save the `priorityFee` from `priorityFee, baseFee = SplitGas()` into `actionCtx`?
then it will be much easier here to collect the priority fee and add up